align/primitive/default/mos.py

```diff
@@ -22,13 +22,12 @@ def __init__(self, pdk, height, fin, gate, gateDummy, shared_diff, stack, bodysw
         self.gate = (2*gate)*self.stack
         self.gatesPerUnitCell = self.gate + 2*self.gateDummy*(1-self.shared_diff)
         self.finDummy = (self.finsPerUnitCell-fin)//2
-        self.lFin = 16 ## need to be added in the PDK JSON
+        self.lFin = height ### This defines numebr of fins for tap cells; Should we define it in the layers.json?
```

Comment:
How are we using it currently? Does every primitive get a tapcell row on top of it?